### ISSUE TITLE
Android > 'No products found' & 'No Categories found' message while selecting an Outlet

### ISSUE THREAD

**Issue by sreeja-eglobeits** [gopinath-eglobeits/ebevco-customer#6]
Android > 'No products found' & 'No Categories found' message while selecting an Outlet (while running the loader)
Categories & Products are displaying after the message

![product](https://user-images.githubusercontent.com/52734274/80587898-ffa29900-8a34-11ea-9a44-b5b01d608d60.JPEG)

![categories](https://user-images.githubusercontent.com/52734274/80587912-03ceb680-8a35-11ea-9523-1420709ad320.JPEG)

**Comment by gopinath-eglobeits** [comment]
Is it really an issue? I guess that is how they generally functions. The list shows only if there is at least one item to show.

**Comment by sreeja-eglobeits** [comment]
As mentioned in the issue description, 'Categories & Products are displaying after the message'

**Comment by gopinath-eglobeits** [comment]
Did you the mean 'NoCategories found' message is showing even after the list is populated?

**Comment by sreeja-eglobeits** [comment]
No. Just before the list display above mentioned message is displaying

**Comment by gopinath-eglobeits** [comment]
That is quite natural. It is an indicator of initial status before fetching and checking whether a list is  present from the admin side. Do you have any other suggestion on how we can handle this?
